src/NodeNameSpace.js

```diff
@@ -248,9 +300,8 @@ x3dom.NodeNameSpace.prototype.setupTree = function (domNode, parent) {
             };
         }
 
-        // TODO (?): dynamic update of USE attribute during runtime
-        if (domNode.hasAttribute('USE') || domNode.hasAttribute('use'))
-        {
+        // TODO: dynamic update of USE attribute during runtime
```

Comment:
Sorry about that. I have added back.

Reply:
While looking through the SAI spec., it appears it could be allowed but the official method is to replace the complete element, not just the USE attribute.